eng/devices/android.cake

```diff
@@ -81,7 +81,8 @@ Setup(context =>
 			else
 				DEVICE_ARCH = "arm64-v8a";
 		}
-		DEVICE_ID = $"system-images;android-{api};google_apis;{DEVICE_ARCH}";
+		var sdk = api >= 24 ? "google_apis_playstore" : "google_apis";
+		DEVICE_ID = $"system-images;android-{api};{sdk};{DEVICE_ARCH}";
```

Comment:
Use GPS emulators on API 24+. I think this is the real "fix". Somehow...

The tests that were failing were the ones checking for ability to open `mailto:`. It was working before. Not it stopped. It might have been that an update to the emulator image got damaged or they removed something from it.